gazelle/python/generate.go

```diff
@@ -551,3 +555,49 @@ func ensureNoCollision(file *rule.File, targetName, kind string) error {
 	}
 	return nil
 }
+
+func generateProtoLibraries(args language.GenerateArgs, pythonProjectRoot string, visibility []string, res *language.GenerateResult) {
+	// First, enumerate all the proto_library in this package.
+	var protoRuleNames []string
+	for _, r := range args.OtherGen {
+		if r.Kind() != "proto_library" {
+			continue
+		}
+		protoRuleNames = append(protoRuleNames, r.Name())
+	}
+	sort.Strings(protoRuleNames)
+
+	// Next, enumerate all the pre-existing py_proto_library in this package, so we can delete unnecessary rules later.
+	pyProtoRules := map[string]bool{}
+	for _, r := range args.File.Rules {
+		if r.Kind() == "py_proto_library" {
+			pyProtoRules[r.Name()] = false
+		}
+	}
+
+	// Now generate a py_proto_library for each proto_library.
+	emptySiblings := treeset.Set{}
+	for _, protoRuleName := range protoRuleNames {
+		pyProtoLibraryName := protoRuleName + "_py_pb2"
```

Comment:
[OK to punt to a followup Issue/PR] Let's expose this suffix as another directive, just like we have `python_library_naming_convention`, `python_binary_naming_convention`, and `python_test_naming_convention`, we should have `python_proto_naming_convention`.

Reply:
Ah, TIL -- this is pretty neat. Yeah, happy to land this as a follow-up.

Reply:
Could you make an Issue (feature request) for this?